test/unit/models/members.test.js

```diff
@@ -0,0 +1,123 @@
+const chai = require("chai");
```

Comment:
Any reason why we are pushing members test as well?

Reply:
ohh sorry for that I by mistakely created branch from tests/members so that why it is showing the changes from that branch also, is there something i can do to solve this problem?